monitor/webserver/nginx_landing_page.sls

```diff
@@ -0,0 +1,60 @@
+
+nginx:
+  pkg:
+    - installed
+
+openssl:
+  pkg:
+    - installed
+
+/usr/share/nginx/html/id:
+  file:
+    - managed
+    - makedirs: true
+    - contents: '{{ grains.id }}'
+    - require:
+      - pkg: nginx
+
+/usr/share/nginx/html:
+  file:
+    - recurse
+    - source: salt://monitor/var/www/html
+    - template: jinja
+    - makedirs: true
+    - require:
+      - pkg: nginx
+
+/etc/nginx/conf.d/:
```

Comment:
Shouldn't this be /etc/nginx/sites-enabled since it's an ubuntu machine?  Or does ubuntu honor conf.d/ too?